gips/0055-chain-integration-process.md

```diff
@@ -0,0 +1,97 @@
+
+# Motivation
+
+Following the [MIPs (Migration Infrastructure Providers) program's](https://thegraph.com/migration-incentive-program/) success, The Graph Foundation proposes a new governance-based approach for new chains to be integrated into the protocol and supported on mainnet. Given the protocol is blockchain-agnostic, and we’re entering a new [world of data services](https://forum.thegraph.com/t/gip-0042-a-world-of-data-services/3761), adopting a more open, transparent, and scalable process is necessary. In addition, as the protocol’s mission is to allow anyone to index and access the world’s data in a decentralized, verifiable, and performant way, onboarding new blockchains must also be as open, decentralized, and transparent as possible. 
+
+For this reason, this GIP proposes a lightweight process targeting teams seeking protocol integration (integrators) for decentralized access to new on-chain data sources. The GIP details how integrators would follow a governance-based 3-step process for the new blockchain to be eligible for indexing rewards on The Graph’s decentralized network. Specific requirements can be found below. 
+
+# High-Level Description
+
+At a high level, the integration happens following a 3-stage process:
+
+1. Teams manifest interest in protocol integration by creating a new thread in [The Graph's Forum](https://forum.thegraph.com/), under `Governance & GIPs` > `New Data Sources` ([direct link](https://forum.thegraph.com/c/governance-gips/new-chain-support/71)).
+2. The integraiton is tested in at least one GUI facilitating subgraph testing and publishing to the network such as [Graph Studio](https://thegraph.com/studio/), and The Graph's testnet.
+3. The team writes a GIP proposing support for indexing rewards on The Graph's decentralized network (mainnet). The Council will review the GIP, taking into account the success of Stage 2 and the Indexer community feedback.
+
+```mermaid
+flowchart LR
+    A["`**Stage 1**
+    Integration proposal via a new Forum thread 
+    `"] --> B("`**Stage 2** 
+    Integration validation on Graph's testnet & Subgraph Studio 
+    (or similar GUI)`")
+    B --> C("`**Stage 3**
+    Council-approved Mainnet integration via GIP
+    (Graph Improvement Proposal)`")
+```
+
+Mainnet support for a new `data source type`, such as a new blockchain, means, in the current state of the protocol, having the chain recognized by the Council, on mainnet, as a new `data source type` with detailed support for rewards and arbitration. This is defined in [GIP-0008](https://snapshot.org/#/council.graphprotocol.eth/proposal/0xbdd884654a393620a7e8665b4289201b7542c3ee62becfad133e951b0c408444) using the protocol’s [Feature Matrix Support](https://github.com/graphprotocol/indexer/blob/main/docs/feature-support-matrix.md). An example of a Council-ratified feature matrix [is here](https://snapshot.org/#/council.graphprotocol.eth/proposal/0xd40fe605ecc3d0faca07d6c8d85a3f0d66106ef9e206aa57397de776f0a76a2c). 
```

Comment:
I think "type" is misleading, as this could just be specification of the new network (i.e. it changes the `network` field on a subgraph, rather than the `kind` of datasource)

Reply:
makes sense. I've just removed `type`